patsy/highlevel.py

```diff
@@ -53,7 +53,7 @@ def _try_incr_builders(formula_like, data_iter_maker, eval_env,
     if isinstance(formula_like, ModelDesc):
         return design_matrix_builders([formula_like.lhs_termlist,
                                        formula_like.rhs_termlist],
-                                      data_iter_maker,
+                                      data_iter_maker, eval_env,
```

Comment:
Another trivial nitpick: I prefer to either put all args on one line, or else put each arg on its own line -- easier to read that way IMO